src/App.tsx

```diff
@@ -10,31 +11,32 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState('tab-1');
+
+  const onTabSelected = (id: string) => {
+    setSelectedTabId(id);
+  };
+
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId);
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is Tab 1
+        Selected tab is
+        {selectedTab?.title}
       </h1>
 
       <div data-cy="TabsComponent">
         <div className="tabs is-boxed">
-          <ul>
-            <li className="is-active" data-cy="Tab">
-              <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-            </li>
-
-            <li data-cy="Tab">
-              <a href="#tab-2" data-cy="TabLink">Tab 2</a>
-            </li>
-
-            <li data-cy="Tab">
-              <a href="#tab-3" data-cy="TabLink">Tab 3</a>
-            </li>
-          </ul>
+          <Tabs
+            tabs={tabs}
+            selectedTabId={selectedTabId}
+            onTabSelected={onTabSelected}
+          />
         </div>
 
         <div className="block" data-cy="TabContent">
-          Some text 1
+          {selectedTab?.content}
```

Comment:
why is content separate from tab?

it can be better 
` <Tabs
        tabs={tabs}
        onTabSelected={handlerSelectTab}
        selectedTabId={selectedTab.id}
      />`